patches/minecraft/net/minecraft/block/BlockChorusFlower.java.patch

```diff
@@ -0,0 +1,19 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockChorusFlower.java
++++ ../src-work/minecraft/net/minecraft/block/BlockChorusFlower.java
+@@ -52,7 +52,7 @@
+             {
+                 int i = ((Integer)p_180650_3_.func_177229_b(field_185607_a)).intValue();
+ 
+-                if (i < 5 && p_180650_4_.nextInt(1) == 0)
++                if (i < 5 &&  net.minecraftforge.common.ForgeHooks.onCropsGrowPre(p_180650_1_, blockpos, p_180650_3_, p_180650_4_.nextInt(1) == 0))
+                 {
+                     boolean flag = false;
+                     boolean flag1 = false;
+@@ -141,6 +141,7 @@
+                     {
+                         this.func_185605_c(p_180650_1_, p_180650_2_);
+                     }
++                    net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.world.BlockEvent.CropGrowEvent.Post(p_180650_1_, p_180650_2_, p_180650_3_, p_180650_1_.func_180495_p(p_180650_2_)));
```

Comment:
@LexManos Didn't think I would need to include an @mention. Using ForgeHooks would save 69 characters for that line.  Do you want me to do that? (I don't know what the threshold for saving size is and am asking for direction, I'd really like to get this merged so I can start using it).